point-to-point/p2pchannel/p2p.go

```diff
@@ -75,12 +76,12 @@ func (channel *P2p) Receive() (bool, string) {
 
 	if ok {
 		return true, string(msg.Body[:])
-	} else {
-		return false, ""
 	}
+
+	return false, ""
```

Comment:
Guidelines favour happy paths being the leftmost code, so advocate against the use of else if/when possible.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow